README.adoc

```diff
@@ -38,24 +41,27 @@ cd docs_dir
 
 . Re-associate your files with the proper repository.
 
-.. _If this project is its own Git repo_, change the project's remote origin Git repository.
+.. _If your docs project is its own Git repo_, change the project's remote origin Git repository.
 +
 ----
 git remote set-url origin git://github.com:your-project/your-repo-name.git
```

Comment:
I think we should use <your-project>/<your-repo-name>.git intead. 
I copied and pasted that bad boy as-is because I think I'm used to seeing things I should replace <like this>.

Reply:
whoa!! so ironic. I was enclosing things in <> and there they go, lol. Trying again....
I think we should use "<your-project>/<your-repo-name>.git"

Reply:
Oh ffs.
Do you know what I'm trying to say?

Reply:
`git://github.com:<your-project>/<your-repo-name>.git`

Reply:
yup, that :)